src/features/registry.js

```diff
@@ -58,4 +58,20 @@ systemJSPrototype.delete = function (id) {
         depLoad.i.splice(importerIndex, 1);
     });
   return delete this[REGISTRY][id];
-};
\ No newline at end of file
+};
+
+systemJSPrototype[Symbol && Symbol.iterator ? Symbol.iterator : '@@iterator'] = function() {
+  const registry = this[REGISTRY]
+  const keys = Object.keys(registry);
+  let index = 0;
+  return {
+    next() {
```

Comment:
One way to write this a little more succinctly might be to skip the branch and just have (again, byte shaving):

```js
const key = keys[index++];
return { done: index > keys.length, value: key && [key, registry[key]] };
```